appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourceConstants.java

```diff
@@ -251,6 +251,7 @@ public final class ResourceConstants {
     public static final String CONTEXT_INFO_ENABLED = "context-info-enabled";
     public static final String THREAD_PRIORITY = "thread-priority";
     public static final String LONG_RUNNING_TASKS = "long-runnings-tasks";
+    public static final String USE_FORK_JOIN_POOL = "use-forkjoinpool";
```

Comment:
This looks like it's meant to align to the dotted-name representation of the config bean, so should be in kebab-case
```suggestion
    public static final String USE_FORK_JOIN_POOL = "use-fork-join-pool";
```